test/liblangutil/Scanner.cpp

```diff
@@ -771,6 +771,65 @@ BOOST_AUTO_TEST_CASE(irregular_line_breaks_in_strings)
 	}
 }
 
+BOOST_AUTO_TEST_CASE(solidity_keywords)
+{
+	// These are tokens which have a different meaning in Yul.
+	string keywords = "return byte bool address var in true false leave";
```

Comment:
Could add `switch`, `case`, `default`, `hex`

Reply:
You're killing me :wink:

Reply:
Cmon let's merge it

Reply:
Actually this triggers more tests to be added, on hex/unicode, so I'm doing that. Unfortunate me.

Reply:
Done.